### ISSUE TITLE
Jakarta EE 11:  Support Core Profile, Web Profile and Platform 11

### ISSUE THREAD

**Issue by jhanders34** [OpenLiberty/open-liberty#25092]
## Description

Implement the Jakarta Platform 11 specification including Core Profile and Web Profile

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
## Documents

When available, add links to required feature documents. Use "N/A" to mark particular documents which are not required by the feature.

- Externally raised requests for enhancements: 
  - [Aha](https://cloud-platform.ideas.aha.io/): Link to the Aha idea (also add a link to this issue in the Aha idea)
    - Feature owner adds label `Aha idea`
  - [Open Liberty Feature Request](https://github.com/OpenLiberty/open-liberty/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=): Link to the OL GH issue (also add a link to this issue in the Feature Request GH issue)
    - Feature owner adds label `Requested feature`
- UFO: https://ibm.box.com/s/99shp7fut4mcvcirhi0t5wyy08ruhlsk
- FTS: https://github.com/OpenLiberty/open-liberty/issues/34602
- Beta Blog: https://github.com/OpenLiberty/open-liberty/issues/34471
- GA Blog: Link to GA Blog Post GH Issue

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
## Process Overview

- [Prioritization](#prioritization)
- [Design](#design)
- [Implementation](#implementation)
- [Legal and Translation](#legal-and-translation)
- [Beta](#beta)
- [GA](#ga)
  - [Focal Point Approvals](#focal-point-approvals-complete-by-feature-complete-date)
- [Other Deliverables](#other-deliverables)

### General Instructions

The process steps occur roughly in the order as presented. Process steps occasionally overlap.

Each process step has a number of tasks which must be completed or must be marked as not applicable ("N/A").

Unless otherwise indicated, the tasks are the responsibility of the Feature Owner or a Delegate of the Feature Owner.

If you need assistance, reach out to the [OpenLiberty/release-architect](https://github.com/orgs/OpenLiberty/teams/release-architect).

**Important: Labels are used to trigger particular steps and must be added as indicated.** 

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
## **Prioritization** (Complete Before Development Starts)

The ([OpenLiberty/chief-architect](https://github.com/orgs/OpenLiberty/teams/chief-architect)) and area leads are responsible for prioritizing the features and determining which features are being actively worked on.

### **Prioritization**
- [x] Feature added to the "New" column of the [Open Liberty project board](https://github.com/orgs/OpenLiberty/projects/2)
  - Epics can be added to the board in one of two ways:
    - From this issue, use the "Projects" section to select the appropriate project board.
    - From the appropriate project board click "Add card" and select your Feature Epic issue
- [x] Priority assigned
  - Attend the Liberty Backlog Prioritization meeting

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
## **Design** (Complete Before Development Starts)

Design preliminaries determine whether a formal design, which will be provided by an [Upcoming Feature Overview (UFO)](https://ibm.box.com/v/UFO-Template) document, must be created and reviewed.  A formal design is required if the feature requires any of the following: UI, Serviceability, SVT, Performance testing, or non-trivial documentation/ID.  Furthermore, each identified item places a blocking requirement on another team so it must be identified early in the process.  The feature owner may check-off the item if they know it doesn't apply, but otherwise they should work with the focal point to determine what work, if any, will be necessary and make them aware of it.

### **Design Preliminaries**
- [x] UI requirements identified, or N/A. (Feature owner and [UI focal point](https://github.com/orgs/OpenLiberty/teams/ui-approvers))
- [x] Accessibility requirements identified, or N/A. (Feature owner and [Accessibility focal point](https://github.com/orgs/OpenLiberty/teams/accessibility-approvers))
- [x] ID requirements identified, or N/A. (Feature owner and [ID focal point](https://github.com/orgs/OpenLiberty/teams/id-approvers))
   - Refer to [Documenting Open Liberty](https://github.com/OpenLiberty/open-liberty/wiki/Documenting-Open-Liberty).
   - Feature Owner adds label `ID Required`, if non-trivial documentation needs to be created by the ID team.
   - ID adds label `ID Required - Trivial`, if no design will be performed and only trivial ID updates are needed.
- [x] Serviceability requirements identified, or N/A. (Feature owner and [Serviceability focal point](https://github.com/orgs/OpenLiberty/teams/serviceability-approvers))
- [x] SVT requirements identified, or N/A. (Feature owner and [SVT focal point](https://github.com/orgs/OpenLiberty/teams/svt-approvers))
- [x] Performance testing requirements identified, or N/A. (Feature owner and [Performance focal point](https://github.com/orgs/OpenLiberty/teams/performance-approvers))

### **Design**
- [x] POC Design / UFO review scheduled.
  - Follow the instructions in POC-Forum repo
- [x] POC Design / UFO review completed.
- [x] POC / UFO Review follow-ons completed.
- [x] POC Design / UFO approval requested.
  - Feature owner adds label `Design Approval Request`
- [x] Design / UFO approved. ([OpenLiberty/chief-architect](https://github.com/orgs/OpenLiberty/teams/chief-architect)) or N/A
  - ([OpenLiberty/chief-architect](https://github.com/orgs/OpenLiberty/teams/chief-architect)) adds label `Design Approved`
  - Add the public link to the UFO in Box to the [Documents](#documents) section.
  - The UFO must always accurately reflect the final implementation of the feature. Any changes must be first approved. Afterwards, update the UFO by creating a copy of the original approved slide(s) at the end of the deck and prepend "OLD" to the title(s). A single updated copy of the slide(s) should take the original's place, and have its title(s) prepended with "UPDATED".

### **No Design**
- [ ] No Design requested.
  - Feature owner adds label `No Design Approval Request`
- [x] No Design / No UFO approved. ([OpenLiberty/chief-architect](https://github.com/orgs/OpenLiberty/teams/chief-architect)) or N/A
  - Approver adds label `No Design Approved`
- [x] Feature / Capability stabilization or discontinuation or N/A
  - Feature owner adds label `Product Management Approval Request` and notifies [OpenLiberty/product-management](https://github.com/orgs/OpenLiberty/teams/product-management)
  - Approver adds label `Product Management Approved` ([OpenLiberty/product-management](https://github.com/orgs/OpenLiberty/teams/product-management))
  - Note: For stabilized, superseded, and discontinued feature/capability, skip the [Beta](#beta) section of the template (you may delete it).  Otherwise, proceed as normal.

### **FAT Documentation**
- [ ] "Feature Test Summary" child task created
  - Use the [Feature Test Summary Template](https://github.com/OpenLiberty/open-liberty/issues/new?assignees=&labels=Feature+Test+Summary&template=feature_test_summary.md&title=)
  - Add FTS issue link to the [Documents](#documents) section.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
## **Implementation**

A feature must be [prioritized](https://github.com/orgs/OpenLiberty/projects/2) before any implementation work may begin to be delivered (inaccessible/no-ship).  However, a design focused approach should still be applied to features, and developers should think about the feature design prior to writing and delivering any code.  
Besides being prioritized, a feature must also be socialized (or No Design Approved) before any beta code may be delivered.  All new Liberty content must be inaccessible in our GA releases until it is [Feature Complete](#feature-complete) by either marking it `kind=noship` or [beta fencing](#beta-code) it.  
Code may not GA until this feature has obtained the `Design Approved` or `No Design Approved` label, along with all other tasks outlined in the [GA](#ga) section.

### **Feature Development Begins**
- [x] Add the `In Progress` label

## **Legal and Translation**

In order to avoid last minute blockers and significant disruptions to the feature, the **legal items need to be done as early in the feature process as possible**, either in design or as early into the development as possible.  Similarly, translation is to be done concurrently with development.  Both **MUST** be completed before Beta or GA is requested.

### **Legal** (Complete before Feature Complete Date)
- [x] Changed or new open source libraries are cleared and approved, or N/A. (Legal Release Services/Cass Tucker/Release PM).

### **Innovation** (Complete 1 week before Feature Complete Date)
- [x] Consider whether any aspects of the feature may be patentable. If any identified, disclosures have been submitted.

### **Translation** (Complete by Feature Complete Date)
- [x] PII (Program Integrated Information) updates are merged (i.e. all English strings due for translation have been delivered), or N/A.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
## **Beta**

In order to facilitate early feedback from users, all new features and functionality should first be released as part of a beta release.

### **Beta Code**
- [x] [Beta fence](https://github.com/OpenLiberty/open-liberty/wiki/Beta-Fencing) the functionality
  - E.g. `kind=beta`, `ibm:beta`, `ProductInfo.getBetaEdition()`
- [x] Beta development complete and feature ready for inclusion in a beta release
  - Add label `target:beta` and the appropriate `target:YY00X-beta` (where YY00X is the targeted beta version).
- [x] Feature delivered into beta
  - ([OpenLiberty/release-manager](https://github.com/orgs/OpenLiberty/teams/release-manager)) adds label `release:YY00X-beta` (where YY00X is the first beta version that included the functionality).

### **Beta Blog** (Complete by beta eGA)
- [x] Beta blog issue created and populated using the [Open Liberty BETA blog post](https://github.com/OpenLiberty/open-liberty/issues/new/choose) template.
  - Add a link to the beta blog issue in the [Documents](#documents) section.
  - Note: This is for inclusion into the overall [beta release blog post](https://openliberty.io/blog/?search=beta&key=tag).  If, in addition, you'd also like to create a dedicated blog post about your feature, then follow the "Standalone Feature Blog Post" instructions under the [Other Deliverables](#other-deliverables) section.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
## **GA**

A feature is ready to GA after it is Feature Complete and has obtained all necessary Focal Point Approvals. 

### **Feature Complete**
- [ ] Feature implementation and tests completed.
  - [ ] All PRs are merged.
  - [ ] All related/child issues are closed.
  - [ ] All stop ship issues are completed.
- [x] Legal: all necessary approvals granted.
- [x] Translation: Feature may only proceed to GA if it has either `Translation - Complete` or `Translation - Missing` label
  - If all translation has been delivered to `release` branch, feature owner adds label `Translation - Complete`.
  - If missing translation does not cause a break in functionality, nor a security or production outage risk, feature owner adds label `Translation - Missing`.
    - Once all missing translations are delivered, the `Translation - Missing` label is replaced with `Translation - Complete`.
  - If missing translation could cause a break in functionality or a security or production outage risk, feature owner adds the `Translation - Blocked` label.
    - Featues with `Translation - Blocked` may **NOT** proceed to GA until the label has been replaced with either `Translation - Missing` or `Translation - Complete`.
  - For further guidance, contact [Globalization focal point](https://github.com/orgs/OpenLiberty/teams/globalization-approvers) or the [Release Architect](https://github.com/orgs/OpenLiberty/teams/release-architect).
- [ ] GA development complete and feature ready for inclusion in a GA release
  - Add label `target:ga` and the appropriate `target:YY00X` (where YY00X is the targeted GA version).
  - Inclusion in a release requires the completion of all Focal Point Approvals.

### **Focal Point Approvals** (Complete by Feature Complete Date)

These occur only after GA of this feature is requested (by adding a `target:ga` label).  GA of this feature may not occur until all approvals are obtained.

### **All Features**
- [ ] **APIs/Externals** - Externals have been reviewed or N/A.  ([OpenLiberty/externals-approvers](https://github.com/orgs/OpenLiberty/teams/externals-approvers))
  - Approver adds label `focalApproved:externals` 
- [ ] **Demo** - Demo is scheduled for an upcoming EOI or N/A. ([OpenLiberty/demo-approvers](https://github.com/orgs/OpenLiberty/teams/demo-approvers))
  - Add comment `@OpenLiberty/demo-approvers Demo scheduled for EOI [Iteration Number]` to this issue.
  - Approver adds label `focalApproved:demo`.
- [ ] **FAT** - All Tests complete and running successfully in SOE or N/A. ([OpenLiberty/fat-approvers](https://github.com/orgs/OpenLiberty/teams/fat-approvers))
  - Approver adds label `focalApproved:fat`.

### **Design Approved Features**
- [ ] **ID** - Documentation is complete or N/A. ([OpenLiberty/id-approvers](https://github.com/orgs/OpenLiberty/teams/id-approvers))
  - Approver adds label `focalApproved:id`.
  - > **_NOTE:_**  If only trivial documentation changes are required, you may reach out to the ID Feature Focal to request a `ID Required - Trivial` label.  Unlike features with regular ID requirement, those with `ID Required - Trivial` label do not have a hard requirement for a Design/UFO.
- [x] **InstantOn** - InstantOn capable or N/A. ([OpenLiberty/instantOn-approvers](https://github.com/orgs/OpenLiberty/teams/instantOn-approvers))
  - Approver adds label `focalApproved:instantOn`.
- [ ] **Performance** - Performance testing is complete or N/A. ([OpenLiberty/performance-approvers](https://github.com/orgs/OpenLiberty/teams/performance-approvers))
  - Approver adds label `focalApproved:performance`.
- [ ] **Serviceability** - Serviceability has been addressed or N/A. ([OpenLiberty/serviceability-approvers](https://github.com/orgs/OpenLiberty/teams/serviceability-approvers))
  - Approver adds label `focalApproved:sve`.
- [ ] **STE** - Skills Transfer Education chart deck is complete or N/A. ([OpenLiberty/ste-approvers](https://github.com/orgs/OpenLiberty/teams/ste-approvers))
  - Approver adds label `focalApproved:ste`.
- [ ] **SVT** - System Verification Test is complete or N/A. ([OpenLiberty/svt-approvers](https://github.com/orgs/OpenLiberty/teams/svt-approvers))
  - Approver adds label `focalApproved:svt`.

### **Remove Beta Fencing** (Complete by Feature Complete Date)
- [ ] Beta guards are removed, or N/A
  - Only after all necessary Focal Point Approvals have been granted.

### **GA Blog** (Complete by Friday after GM)
- [ ] GA Blog issue created and populated using the [Open Liberty GA release blog post](https://github.com/OpenLiberty/open-liberty/issues/new/choose) template.
  - Add a link to the GA Blog issue in the [Documents](#documents) section.
  - Note: This is for inclusion into the overall [release blog post](https://openliberty.io/blog/?search=release&key=tag).  If, in addition, you'd also like to create a dedicated blog post about your feature, then follow the "Standalone Feature Blog Post" instructions under the [Other Deliverables](#other-deliverables) section.

### **Post GM** (Complete before GA)
- [ ] After confirming this feature has been included in the GM driver, feature owner closes this issue.

### **Post GA**
- [ ] Remove the `target:ga` and `target:YY00X` labels, and add the appropriate `release:YY00X`. ([OpenLiberty/release-manager](https://github.com/orgs/OpenLiberty/teams/release-manager))

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
## **Other Deliverables**

- [ ] **Standalone Feature Blog Post** - A blog post specifically about your feature or N/A.  (Feature owner and [OpenLiberty/release-architect](https://github.com/orgs/OpenLiberty/teams/release-architect))
  - This should be strongly considered for larger or more prominent features.
  - Follow [instructions](https://github.com/OpenLiberty/blogs/tree/draft#writing-and-publishing-blog-posts-on-the-openlibertyio-blog) in the blogs repo.
- [ ] **OL Guides** - OL Guides assessment is complete or N/A. ([OpenLiberty/guide-assessment](https://github.com/orgs/OpenLiberty/teams/guide-assessment/members))
- [ ] **Dev Experience** - Developer Experience & Tools work is complete or N/A. ([OpenLiberty/dev-experience-assessment](https://github.com/orgs/OpenLiberty/teams/dev-experience-assessment/members))
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

**Comment by neuwerk** [comment]
UFO Socialization Notes

Slide 10:

- [x] `data-1.0 will be included in the webProfile-10.0 feature` should be updated to `data-1.0 will be included in the webProfile-11.0 feature`

- [x] Should we drop xmlWS-4.0 support in the app client? No clear decision on this, might be a moot point if app client support is dropped completely in EE11
  - Stated that it would be removed from jakartaeeClient-11.0 and removed the thought of xmlWSClient-4.0 additional feature.

Slide 12:

- [x] Add `globalHandler` feature to list of protected features to be updated

- [x] `springBoot-3.0` - we should make it work as the spring community has plans to make it work with ee11. 

Slide 16:

- [x] Bullet point 3 - it's `usageMetering-1.0` not `usagingMetering-1.0`

Slide 17:

- [x] Add a graphical representation of end user overview 

Slide 20:

- [x] Managed beans should be mentioned as well as in the feature design slides. 

- [x] Mention the newly suggested API for obtaining current versions of the specs that are available on the runtime

Slide 23:

- [x] Added to Open Liberty Starter as the default  
  - Also added the Jakarta EE starter to the slide

Slide 28:

- [x] Should we let individual specs be beta? Yes, each component spec should be beta as soon as it can show new EE 11 functionality.
  - Added a statement to make this clear

Slide 32:

- [x] Should we keep optional features in zips? Yes, we should keep Jakarta EE Optional Features in Zips

- [x] Remove this line
  `Not creating a Jakarta EE 11 archive install that includes Semeru Certified Edition 21. A future Install Epic could add this ship vehicle for Semeru 21 for Windows and other platforms.`
  - I added the requirement to have the ability to install Semeru 21 with Installation Manager and having Liberty container images being available on top of Semeru 21 container images in order to ship Jakarta EE 11 support.

Additional comments: 

- [x] Should mention support/testing with instantOn when it's ready.
  - Added a line to the Automated Testing slide and the Cloud considerations slide

**Comment by njr-11** [comment]
Review comments from part 2 of the Upcoming Feature Overview:

- [x] Slide 6 - possibly list EJB 2.0 portions of EJB spec as removed
- [x] Slide 8 - specifically mention MicroProfile versions
- [x] Slide 14 - Update MicroProfile section to spell out what was already done for MP 7 and 7.1
- [x] Slide 30 - mention additional features such as xml web services
- [ ] Slide 41 - have offline discussion of migration with respect to optional features

**Comment by ayoho** [comment]
## Meeting Summary — February 27, 2026
- Jakarta EE Platform, Web Profile & Core Profile (Part 2) — Open Liberty Epic 25092

---

**Optional features removed from EE 11 platform (Emily Jiang):**
- Emily asked Jared to list out which optional features were removed from the EE 11 platform. Jared jumped to the relevant slide showing three specs no longer in the platform. He also noted that EJB 2.0 (an optional part of the EJB spec, not a standalone optional spec) could be added to that list.
- Nathan noted he'd make a note of that.

**MicroProfile version compatibility with EE 11 (Emily Jiang):**
- Emily asked which MicroProfile versions had been tested with EE 11, expecting MicroProfile 7.0 and 7.1 to both work. Jared confirmed both were tested and work. Emily asked that the slide be updated to call out specific versions rather than saying "future."
- Emily noted that backward-incompatible changes in the EE 11 Core Profile do not impact MicroProfile APIs, so compatibility holds.
- Emily also asked about the packaging policy for MicroProfile in the Liberty zip — Jared confirmed all compatible versions are packaged (both 7.0 and 7.1 are included in the Jakarta EE 11 WebSphere Liberty package). Emily said that was better than she expected.

**XML Web Services / XML Binding no longer in the EE 11 convenience feature (Chuck Bridgham, Emily Jiang):**
- Chuck asked whether XML Web Services would be included in the Jakarta EE 11 platform (versionless feature). Jared confirmed that if a user specifies `xmlWS` or `xmlBinding` (JAX-WS / JAX-B), the versionless feature will enable it — but these are no longer automatically pulled in by the convenience feature.
- Chuck noted this is a meaningful difference from prior behavior and suggested it be called out on the relevant slide. Jared agreed and said he'd add a note, including for Enterprise Beans Home.
- Emily raised a migration concern: users migrating from EE 10 to EE 11 using the umbrella/convenience feature who also use XML Web Services will find those APIs missing and need to explicitly add the feature. She asked whether the migration tooling (AMA / binary scanner) would handle this.
- Chuka explained that the binary scanner scans the application to recommend features, not the existing server configuration, and that it starts from scratch on recommendations — so it should be able to detect XML Web Services usage and recommend the feature. However, Chuka acknowledged there may be some nuance around the Devtools perspective. Emily agreed to track this offline with Chuka.

**EE 11 migration tooling timeline (Emily Jiang, Chuka Obinabo):**
- Emily asked when the EE 11 migration support in AMA would be available, noting that Liberty won't be listable as an EE 11 option in AMA's dropdown until that work is done.
- Chuka confirmed the EE 11 migration work was started last year but paused in favor of the Java 25 work. Once EE 11 support lands in Liberty, the migration work will resume and should be ready by Q3.
- Jared noted this is similar to how Java 25 support in AMA lags Liberty by a quarter or two.

**Nathan's closing notes for Presentation 2:**
- Slide 6: possibly list EJB 2.0 portions of the EJB spec as removed.
- Slides 8 and 14: specifically mention MicroProfile versions (7.0 and 7.1).
- Slide 30: mention additional features such as XML Web Services that must be explicitly added when using the convenience feature.

**Comment by jhanders34** [comment]
**Serviceability Approval Comment** - Please answer the following questions for Serviceability approval:

1.  Have the most likely customer‑facing problem scenarios identified in the Serviceability section of the UFO been addressed in the implementation?

- Slide 38 and 39 of [the UFO](https://ibm.ent.box.com/file/1324731033111?s=99shp7fut4mcvcirhi0t5wyy08ruhlsk) talk about serviceability related messaging related to this feature.
  - Messaging is consistent for all feature that do not start due to conflicts or not being supported with Jakarta EE 11.  That is not changed and behave the same for EE 11 features as previous versions.  `io.openliberty.jakartaee11.internal.tests.EE11FeatureCompatibilityTest` test in the `io.openliberty.jakartaee11.internal_fat` project validates that the feature start or fail to start as expected and get the expected EE mismatch errors saying that there is a conflict or is not supported with Jakarta EE 11
  - New message was added when a server is started with Jakarta EE 11 with Java security enabled.  The SecurityManager is set initially, but when activating with Jakarta EE 11 features, the SecurityManager is disabled and a warning message is output stating that the SecurityManager was stopped.  If switching to non EE 11 features after using EE 11 features, the SecurityManager is re-enabled and an informational message is output explaining that the SecurityManager is now active again.

2.  Test and Demo -- As part of the serviceability process we're asking feature teams to test and analyze common problem paths for serviceability and demo those problem paths to someone not involved in the development of the feature (eg. IBM Support, test team, or another development team).  
a) What problem paths were tested and demonstrated? 
b) Who did you demo to?
c) Do the people you demo'd to agree that the serviceability of the demonstrated problem scenarios is sufficient to avoid Support Cases for any problems customers are likely to encounter, or that IBM Support should be able to quickly address those problems without need to engage SMEs?
- Started with a combination of EE10 and EE11 features and showed that no features started because of the conflict.
- Started server with couchdb-1.0 with EE11 features and showed that it didn't start and indicated the couchdb-1.0 was not compatible with Jakarta EE 11.  This stabilized feature is an example of feature doesn't depend on Jakarta EE technologies, but still worked with EE 10.  It was updated to not work with EE 11.  Again no feature start due to the conflict.
- Started a server with SecurityManager enabled with Jakarta EE 11 features and demonstrated the message that the SecurityManager is stopped and then when adding non Jakarta EE 11 features to the server.xml that it was restarted.

- Demoed to Tracy Burroughs and he agreed that the serviceability is sufficient for the demonstrated problem scenarios. 

3.  Which IBM Support / SME queues will handle Support Cases for this feature?  Ensure they are present in the contact reference file and in the Queue Contact Summary, and that the respective IBM Support/SME teams know they are supporting it.  Ask Prashanth Gunapalasingam if you need links or more info.

- Same teams that support Jakarta EE 10 features will support the Jakarta EE 11 features

4. Does this feature add any new metrics or emit any new JSON events?  If yes, have you updated the JMX metrics reference list / Metrics reference list / JSON log events reference list in the Open Liberty docs?

- No

**Comment by jhanders34** [comment]
@OpenLiberty/ste-approvers STE is available [here](https://ibm.ent.box.com/file/2206563791741).

**Comment by gnadell** [comment]
> [@OpenLiberty/ste-approvers](https://github.com/orgs/OpenLiberty/teams/ste-approvers) STE is available [here](https://ibm.ent.box.com/file/2206563791741).

Thanks for the STE @jhanders34 . STE is approved now.

**Comment by chirp1** [comment]
Ram completed the global differences updates at the following URL: https://github.com/OpenLiberty/docs/issues/8060

Approving
